Source/Particles/ERFPCInitializations.cpp

```diff
@@ -35,6 +59,8 @@ void ERFPC::InitializeParticles (const std::unique_ptr<MultiFab>& a_height_ptr)
 
     if (m_initialization_type == ERFParticleInitializations::init_default) {
         initializeParticlesDefault( a_height_ptr );
+    } else if (m_initialization_type == ERFParticleInitializations::init_box) {
```

Comment:
I think we can get rid of `initializeParticlesDefault()` and consequently, `initializeParticlesDefaultTracersWoA()` and `initializeParticlesDefaultHydro()`. They can now be realized by `initializeParticlesUniformDistributionInBox()` (and they will anyway not be used). 

The default initialization can then be `initializeParticlesUniformDistributionInBox()`. If needed, the initial `particle_box` can be set to the appropriate values if the particle species name is `tracer_particles` or `hydro_particles` (but I wouldn't bother).

Reply:
I'm going to leave this to you to clean up as you see fit ... do whatever makes most sense for your targets :-)